pulse/inngest-router/README.md

```diff
@@ -0,0 +1,130 @@
+# Prisma Pulse-Inngest Router
+
+This package enables event-driven support for your existing Inngest workflows,
+using Prisma Pulse. Trigger workflows instantly whenever a database table
+changes, so you can create timely workflows that run instantly -- not only
+batch workflows.
+
+## Pre-requisites
+
+- A **Pulse API key** which you can get by enabling Pulse in a project in your
+  [Prisma Data Platform](https://pris.ly/pdp) account (learn more in the
+  [docs](https://www.prisma.io/docs/platform/concepts/environments#api-keys))
+- An existing [Inngest](https://www.inngest.com) account and
+  [Event Key](https://www.inngest.com/docs/events/creating-an-event-key?ref=environment-variables)
+
+## Get started
+
+### Fork or clone this repo
+
+You should operate with your own version of this service in a GitHub repository
+that you own. This allows you to track changes, and deploy to cloud platforms
+easily.
+
+### Set up your API keys
+
+Make a copy of the `.env.example` file and update it with your own keys and
+database connection string:
+
+```bash
+cp .env.example .env
+```
+
+### Add your Prisma schema
+
+Add your existing Prisma schema file to the `/prisma` directory, overwriting
```

Comment:
This is a bit confusing to me. What if I don't have a Prisma Schema and I'm just trying things out?

Reply:
That's a very fair point. The original intent of this example was to be a deployable package for existing projects (just for context) which may explain a couple of the things.

But I think with some better explaining it should be OK